core/src/main/java/org/apache/accumulo/core/file/rfile/RFile.java

```diff
@@ -1069,7 +1069,7 @@ private void _seek(Range range) throws IOException {
 
       if (metricsGatherer != null) {
         metricsGatherer.startLocalityGroup(rk.getKey().getColumnFamily());
-        metricsGatherer.addMetric(rk.getKey(), val);
```

Comment:
See above. The gatherer might use it.

Reply:
Fixed in c8301f2a4fa86bee62bd9c5e134add6de13091a7